code/modules/mob/living/carbon/xenobiological/life.dm

```diff
@@ -25,7 +25,7 @@
 		var/turf/heat_turf = get_turf(src)
 		loc_temp = heat_turf.temperature
 	else if(istype(loc, /obj/machinery/atmospherics/unary/cryo_cell))
-		loc_temp = loc:air_contents.temperature
+		loc_temp = loc:air_contents:temperature
```

Comment:
one colon is bad, two is twice as bad, whats the need for this?

Reply:
Because DM_language don't understand what after :